_observing-your-data/ad/index.md

```diff
@@ -119,50 +119,41 @@ After you define the detector, the next step is to configure the model.
 
 ## Step 2: Configure the model
 
-1. Add features to your detector.
+Add features to your detector. A _feature_ is any field in your index that you want to analyze for anomalies. A detector can discover anomalies across one or more features. 
```

Comment:
I think current description is not accurate. A feature can be not only an aggregation of a field but also a painless script.